alex/applications/PublicTransportInfoCS/data/train_names.expanded.txt

```diff
@@ -0,0 +1,178 @@
+albert einstein	albert einstein
```

Comment:
We agreed we will keep the list in the base form in the repository. Not the expanded version.